addons/ai/functions/fnc_unpackStaticWeapon.sqf

```diff
@@ -0,0 +1,132 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Unpacks a static weapon from units' backpacks.
+ *
+ * Arguments:
+ * 0: Gunner <OBJECT>
+ * 1: Assistant <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_gunner, _assistant] call zen_ai_fnc_unpackStaticWeapon
+ *
+ * Public: No
+ */
+
+params ["_gunner", "_assistant", ["_targetPos", [], [[]], 3]];
+//if !(local _gunner) exitWith {_this remoteExecCall [FUNC(unpackStaticWeapon), _gunner]};
```

Comment:
Remove unused code. And in all other places too.

Reply:
https://github.com/ampersand38/ZEN/commit/b2321f23eb771bff8bf96650d34d8b53e0c8978b